app/services/status-messages.ts

```diff
@@ -29,15 +29,15 @@ export default class StatusMessages extends Service {
         // decode commas and \\
         let status = cookies.read(config.OSF.statusCookie).replace(/\\054/g, ',').replace(/\\/g, '');
         // remove outside quotes
-        status = status.slice(1).slice(0, -1);
+        status = status.slice(1, -1);
         let messages = JSON.parse(status);
         messages = messages.map(each => {
             const updated = each;
             updated.id = `status.${each.id}`;
             return updated;
         });
         cookies.clear(config.OSF.statusCookie, {
-            domain: config.environment === 'development' ? 'localhost' : '.osf.io',
+            domain: window.location.hostname,
```

Comment:
Needs to come from the config so that it’s overridable. Hostname won’t work either because we use .osf.io

Reply:
will put it into config! However, we don't use .osf.io for the cookies being created by flask response .set_cookie